### PR TITLE
To Do list: add, remove and interactive list

### PR DESCRIPTION
**Pull request by lilskyex0x**
### To-Do list
🙇‍♂️Kindly give your code review to this repository.

**These checklists are for the milestone II add and remove project.**
- [ ] Removed all hardcoded items from the tasks array.
- [ ] Created a new JavaScript file for the new functionality.
- [ ] Implemented a function for adding a new task (add a new element to the array).
- [ ] Implemented a function for deleting a task (remove an element from the array).
- [ ] Implemented a function for editing task descriptions.
- [ ] All changes to the To Do List are saved in local storage.
________________
**These checklists are for the milestone III interactive list project.**
- [ ] Added a new JavaScript file and import it as a module:
- [ ] it contains methods related to the status updates (completed: true/false).
- [ ] Added event listener to the checkbox.
- [ ] Updated items object's value for completed key upon user actions.
- [ ] Implemented a function for the "Clear all completed" button (used filter() method).
- [ ] Stored the updated array of items in local storage, so the user gets the correct list values after the page reloads.

❤Thank you!

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (HAMMAS-SALEEM) [inline .vscode/settings.json:1]: image

- [x] You didn't acknowledge anyone in your readme file. :fearful: Kindly add Microverse in this section to acknowledge Microverse :+1:

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Submission :red_circle:

## Hey @lilskyex0x :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following the GitHub flow
✔️ Following some requirements

Good job on implementing some of the requirements but I cannot proceed review of your pull request because there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @lilskyex0x, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️
- Js filter method ✔️
 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .vscode/settings.json (hunk 1 above)
Comment (lilskyex0x) [reply .vscode/settings.json:1]: I will add it right away sir thank you

**Review by muneebulrehman — APPROVED** [review]
Hi @lilskyex0x , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
-  Descriptive PR
- All requirements met for this milestone.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**